packages/benchmark/src/testExecution/k6Executor.ts

```diff
@@ -0,0 +1,28 @@
+import { $ } from 'zx';
+import { TestScenario } from '@/types/testScenario';
+
+/**
+ * Executes test scenarios using k6
+ */
+export class K6Executor {
+	constructor(
+		private readonly k6ExecutablePath: string,
+		private readonly n8nApiBaseUrl: string,
+	) {}
+
+	async executeTestScenario(testCase: TestScenario) {
+		// For 1 min with 5 virtual users
```

Comment:
What does virtual user mean in this context? Haven't looked into k6 yet.

Reply:
It's a single thread/executor/worker that executes the default function exported from k6 script. In practice how many concurrent requests we are sending